lib/src/components/NxTag/NxTag.scss

```diff
@@ -33,26 +34,32 @@
 
 .nx-tag--selectable {
   cursor: pointer;
-  padding-right: var(--nx-spacing-2x);
+  padding-right: calc(var(--nx-spacing-2x) - calc(var(--nx-border-width) - 1px));
```

Comment:
Why the `-1px`?

Reply:
> Why the `-1px`?

To account for dark-mode borders being 2px compared to 1px on light-mode.

Reply:
> To account for dark-mode borders being 2px compared to 1px on light-mode.

I'm afraid I don't follow; could you elaborate? In sketch, I see the following:
![image](https://user-images.githubusercontent.com/3630091/235453041-013476f7-3ed7-4447-a00e-e08ec43aa897.png)
![image](https://user-images.githubusercontent.com/3630091/235453109-6683813c-010c-46cc-a8ad-b47e9020094b.png)

That is, 8px from the edge of the icon to the _outside_ of the tag border.  With your implementation, I'm seeing
![image](https://user-images.githubusercontent.com/3630091/235453210-b518883b-f50e-4e7a-9568-64849e5111c1.png)
in dark mode and
![image](https://user-images.githubusercontent.com/3630091/235453251-ea6c1de4-1a3c-4c49-b05d-4c5380d754d9.png)
 in light mode, so 9px from the edge of the icond to the outside of the tag border.

Reply:
> > To account for dark-mode borders being 2px compared to 1px on light-mode.
> 
> I'm afraid I don't follow; could you elaborate? In sketch, I see the following: 
> 
> That is, 8px from the edge of the icon to the _outside_ of the tag border. With your implementation, I'm seeing ! in light mode, so 9px from the edge of the icond to the outside of the tag border.

Oh yes, you're right, I've changed it here and updated both light mode and dark mode snapshots.
142c4fb6e649203f55b4538522f9e556bd38798a